dist/leaflet.distortableimage.js

```diff
@@ -637,29 +637,14 @@ var EditOverlayAction = L.ToolbarAction.extend({
 		}
 	});
 
-L.DistortableImage.EditToolbar = L.Toolbar.Popup.extend({
-	options: {
-		actions: [
-			ToggleTransparency,
-			RemoveOverlay,
-			ToggleOutline,
-			ToggleEditable,
-			ToggleRotateDistort
-		]
-	},
-
-	/* Remove the toolbar after each action. */
-	_getActionConstructor: function(Action) {
-		var A = Action.extend({
-			removeHooks: function() {
-				var map = this._map;
-
-				map.removeLayer(this.toolbar);
-			}
-		});
-
-		return L.Toolbar.prototype._getActionConstructor.call(this, A);
-	}
+L.DistortableImage.EditToolbar = new LeafletToolbar.Popup({
```

Comment:
This should be `LeafletToolbar.Popup.extend` rather than `new Leaflet.Toolbar.Popup`.